config_and_run/run_in_docker.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+CSV_NAME=$1
+CONFIG_NAME=obscuritext_configs
+
+# Copies csv file and config files into docker
+docker cp $CSV_NAME.csv e37dba6a70c3:/obscuritext-develop/$CSV_NAME.csv
```

Comment:
use double quotes and curly braces when calling bash variable names, e.g. `$CONFIG_NAME` -> `"${CONFIG_NAME}"`